docs/sources/setup/migrate/ssd-to-distributed/_index.md

```diff
@@ -0,0 +1,372 @@
+             period: 24h
+     storage_config:
+       aws:
+         region: eu-central-1
+         bucketnames: aws-chunks-bucket
+         s3forcepathstyle: false
+     ingester:
+       chunk_encoding: snappy
+       # Add this to ingester; this will force ingesters to flush before shutting down
+       wal:
+         flush_on_shutdown: true
+     ruler:
+       enable_api: true
+       storage:
+         type: s3
+         s3:
+           region: eu-central-1
+           bucketnames: aws-ruler-bucket
+           s3forcepathstyle: false
+         alertmanager_url: http://prom:9093
+     querier:
+       max_concurrent: 4
+
+     storage:
+       type: s3
+       bucketNames:
+         chunks: "aws-chunks-bucket"
+         ruler: "aws-ruler-bucket"
+       s3:
+         region: eu-central-1
+
+   # Important: Make sure to change this to SimpleScalable<->Distributed
+   deploymentMode: SimpleScalable<->Distributed
+
+   # SSD
+   backend:
+     replicas: 2
+   read:
+     replicas: 3
+   write:
+     replicas: 3
+
+   # Distributed Loki
+   # Spin up the distributed components
+   ingester:
+     replicas: 3
+     zoneAwareReplication:
+       enabled: false
+   querier:
+     replicas: 3
+     maxUnavailable: 0
+   queryFrontend:
+     replicas: 2
+     maxUnavailable: 0
+   queryScheduler:
+     replicas: 2
+   distributor:
+     replicas: 2
+     maxUnavailable: 0
+   compactor:
+     replicas: 1
+   indexGateway:
+     replicas: 2
+     maxUnavailable: 0
+   ruler:
+     replicas: 1
+     maxUnavailable: 0
+
+   # Single binary Loki
+   singleBinary:
+     replicas: 0
+
+   minio:
+     enabled: false
+   ```
+
+   Here is a breakdown of the changes:
+   * `ingester.wal.flush_on_shutdown: true`: This will force the ingesters to flush before shutting down. This is important to prevent data loss.
+   * `deploymentMode: SimpleScalable<->Distributed`: This will allow for the SSD and distributed components to run simultaneously.
+   * Spin up all distributed components with the desired replicas.
+
+1. Deploy the distributed components using the following command:
+
+    ```bash
+    helm upgrade --values values-migration.yaml loki grafana/loki -n loki 
+    ```
+   {{< admonition type="caution" >}}
+   It is important to allow all components to fully spin up before proceeding to the next stage. You can check the status of the components using the following command:
+
+    ```bash
+    kubectl get pods -n loki
+    ```
+   Let all components reach the `Running` state before proceeding to the next stage.
+   {{< /admonition >}}
+
+## Stage 2: Transitioning to Distributed components
+
+The final stage of the migration involves transitioning all traffic to the distributed components. This is done by scaling down the SSD components and swapping the `deploymentMode` to `Distributed`. To do this:
+
+1. Create a copy of `values-migration.yaml` and name it `values-distributed.yaml` (Once again this can all be done in the original file).
```

Comment:
Just noticing that there is not a command to apply this new file after it has been modified.